cli/tests/test_upgrade_staged_resolver.py

```
@@ -398,6 +402,26 @@ def test_explicit_final_target_still_resolves_verified_two_hop_plan(
     assert "/releases/download/0.8.4/upgrade-manifest.json" in downloads
 
 
+@pytest.mark.parametrize("current_version", ("0.8.3", "0.7.1"))
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list`

Use `list` for parameter values

(PT007)